Libplanet/Net/Transports/TransportException.cs

```diff
@@ -0,0 +1,32 @@
+#nullable enable
+using System;
+using System.Runtime.Serialization;
+
+namespace Libplanet.Net.Transports
+{
+    [Serializable]
+    public class TransportException : Exception
+    {
+        public TransportException()
+        {
+        }
+
+        public TransportException(string message)
+            : base(message)
+        {
+        }
+
+        public TransportException(string message, Exception innerException)
+            : base(message, innerException)
+        {
+        }
+
+        protected TransportException(
+            SerializationInfo info,
+            StreamingContext context
+        )
+            : base(info, context)
+        {
+        }
```

Comment:
Could you add unit test for serialize, deserialize, like [Libplanet.Tests/Blocks/InvalidBlocckTxHashExceptionTest.cs](https://github.com/planetarium/libplanet/blob/2301630e615ab2ac301a4babe3a3012ffc46fa4a/Libplanet.Tests/Blocks/InvalidBlocckTxHashExceptionTest.cs) does?

Reply:
Great, I'll amend it.